help-zh-CN.asc

```diff
@@ -0,0 +1,37 @@
+:docinfo1:
+= AnkiDroid Support
+:sectanchors:
+
+[.text-right.big]
+link:help-ja.html[日本語]
+
+== 提问之前
+
+请求帮助之前, 请先试着先通过这个链接link:manual.html[AnkiDroid 手册]搜索类似问题,你也可以通过  https://github.com/ankidroid/Anki-Android/wiki/FAQ[常见问题解答]列表, 以及官方 link:http://ankisrs.net/docs/manual.html[Anki手册] 查找常见的Anki系统问题，这里的阐述的anki的特点不仅限于android，它适用于各种anki软件系统.
+
+== 支持
+如果你不能从上面提供的文档中找到你所需要的，请进入下面相关网站：
+
+非android特定问题 :: AnkiDroid (Anki的android版本) 不同于其他版本，被单独创建和管理的. Android版本不负责PC / Web / iOS 版本, 以及一般的 Anki 问题, 这些版本遇到的问题请参阅 https://anki.tenderapp.com/[Anki 支持网站].
+
+AnkiDroid问题 :: 有关AnkiDroid的帮助和支持, 请参阅 https://groups.google.com/forum/#!forum/anki-android[用户论坛], 或者你你不想再论坛上注册，你也可以将你的问题发送到这个论坛的 mailto:public-forum@ankidroid.org[公共邮箱] .
```

Comment:
@timrae 
good idea, Ankichina forum has just created, it can provide a communication space for the user to discuss the problem.